kolla/node_custom_config/manila.conf

```diff
@@ -0,0 +1,9 @@
+[DEFAULT]
+{% if default_share_type is defined %}
```

Comment:
`is defined` will also be true if it's empty, I think? You can do `is defined and not empty` I think.